README.md

```diff
@@ -4,21 +4,44 @@
 
 > This JavaScript capstone project is built on the [TVMAZE API](https://static.tvmaze.com/apidoc/), an external API. The cards display the movies information that was retrieved from the API. The user has the option to like and comment on the movies cards.
 
+![](src/assets/cinema.JPG)
+
+![](src/assets/mobil.JPG)
+
 ## Built With
 
-- HTML, CSS, JavaScript, and API
+- HTML, CSS, JavaScript, and Tv-Maze API
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://natanima.github.io/Javascript-capstone-project/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 To get a local copy up and running follow these simple example steps.
 
+
+```
+git clone https://github.com/NatanimA/Javascript-capstone-project.git
+```
+
+```
+npm install
+```
+
+```
+npm run build
+```
+
+Lastly
+
+```
+npm start
+```
+
+😉 
+
 ### Prerequisites
 
 ### Setup
```

Comment:
- Please, let's make our readme more professional by removing the following headers since we've already added the contents above, or kindly move the corresponding contents to it appropriate headers:
 `Setup, Install, Usage, Run test, Deployment`